src/qml/controls/BlockClock.qml

```diff
@@ -0,0 +1,157 @@
+    }
+
+    Behavior on arcEnd {
+       id: animationArcEnd
+       enabled: true
+       NumberAnimation {
+            easing.type: Easing.Bezier
+            easing.bezierCurve: [0.5, 0.0, 0.2, 1, 1, 1]
+            duration: root.animationDuration
+        }
+    }
+
+    Canvas {
+        id: canvas
+        anchors.fill: parent
+        rotation: -90
+
+        onPaint: {
+            var ctx = getContext("2d")
+            var x = width / 2
+            var y = height / 2
+
+            ctx.reset()
+
+            // Paint background
+            ctx.beginPath();
+            ctx.arc(x, y, (width / 2) - parent.lineWidth / 2, 0, Math.PI * 2, false)
+            ctx.lineWidth = root.lineWidth
+            ctx.strokeStyle = root.colorBackground
+            ctx.stroke()
+
+            if (!synced) {
+                var start = Math.PI * (parent.arcBegin / 180)
+                var end = Math.PI * (parent.arcEnd / 180)
+                // Paint foreground arc
+                ctx.beginPath();
+                ctx.arc(x, y, (width / 2) - parent.lineWidth / 2, start, end, false)
+                ctx.lineWidth = root.lineWidth
+                ctx.strokeStyle = root.colorCircle
+                ctx.stroke()
+            }
+
+            else {
+                var del = 0.0025
+                // Paint Block time points
+                for (var i=1; i<parent.blockList.length - 1; i++) {
+                    var starts = Math.PI * ((parent.blockList[i]) * 360 / 180)
+                    var ends = Math.PI * ((parent.blockList[i + 1]) * 360 / 180)
+                    var conf = blockList.length - i - 1
+                    ctx.beginPath();
+                    ctx.arc(x, y, (width / 2) - parent.lineWidth / 2, starts, ends, false)
+                    ctx.lineWidth = root.lineWidth
+                    ctx.strokeStyle = conf > 5 ? colorList[5] : colorList[conf]
+                    ctx.stroke()
+
+                    // Paint dark segments
+                    var start = Math.PI * ((parent.blockList[i + 1] - del) * 360 / 180)
+                    ctx.beginPath();
+                    ctx.arc(x, y, (width / 2) - parent.lineWidth/2, start, ends, false)
+                    ctx.lineWidth = 4
+                    ctx.strokeStyle = root.colorBackground
+                    ctx.stroke();
+                }
+
+                // Print last segment
+                var starts = Math.PI * ((parent.blockList[parent.blockList.length - 1]) * 360 / 180)
+                var ends = Math.PI * (ringProgress * 360 / 180)
+
+                ctx.beginPath();
+                ctx.arc(x, y, (width / 2) - parent.lineWidth / 2, starts, ends, false)
+                ctx.lineWidth = root.lineWidth
+                ctx.strokeStyle = colorList[0];
+                ctx.stroke()
+            }
+        }
+    }
+
+    ColumnLayout {
+        anchors.centerIn: root
+        Image {
+            Layout.alignment: Qt.AlignCenter
+            source: "image://images/app"
```

Comment:
This should be using the bitcoin circle icon introduced in https://github.com/bitcoin-core/gui-qml/pull/176. That's what is specified in the [design files](https://www.figma.com/file/GaCoOSNHB2yMB9ThiDtred/Bitcoin-Core-App?node-id=3259%3A73676&t=pjV9bY1q9KckaP4Q-0)

Reply:
Addressed in 0f5461e.